cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/AnalyzerConfig.java

```diff
@@ -642,6 +653,11 @@ public static ConfigDef define(ConfigDef configDef) {
                             DEFAULT_OPTIMIZATION_OPTIONS_GENERATOR_CLASS,
                             ConfigDef.Importance.LOW,
                             OPTIMIZATION_OPTIONS_GENERATOR_CLASS_DOC)
+                    .define(RACK_AWARE_GOAL_RACK_ID_MAPPER_CLASS_CONFIG,
```

Comment:
Also add the configure in Configurations.md

Reply:
Added doc